AAI/AAIConnectProfile.md

```diff
@@ -429,6 +436,11 @@ the Broker.
                 any other Broker involved in the propagation of the claims to
                 also be trusted if the Claim Clearinghouse needs to restrict its
                 trust model).
+            2.  If an Embedded Token contains an `aud` claim, Clearinghouse MUST check
+                that one of the Embedded Token's `aud` entries 
+                [matches](https://tools.ietf.org/html/rfc3986#section-6.2.1) the Broker's 
```

Comment:
Why broker's? The "aud" claim means "audience", so it should match the clearing house's client id, not the broker's issuer id. The audience ("the recipients processing the JWT" as per [RFC 7519#4.1.3](https://tools.ietf.org/html/rfc7519#section-4.1.3)) is the clearing house, not the broker.

Reply:
Please see the PR summary for what I am trying to achieve.

In addition to allowing the `aud` claim to contain brokers we could say it must also contain clearing houses then the clearing house must verify that it is in the list itself as well as the broker. Essentially the visa issuer would specify all the parties, brokers and clearing houses, who are allowed to use the JWT. This spec would then go above and beyond the JWT RFC by saying the clearing house much check that the broker is in the `aud` claim.

If `aud` is not the right claim to use then I would propose adding a new claim to the spec and registering that with IANA.

Reply:
Defining a new claim seems to be a better way to me.